Index: toys_inventory/toys_inventory/src/main/java/com/example/toys_inventory/ToysInventoryApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.toys_inventory;\r\n\r\nimport org.springframework.boot.SpringApplication;\r\nimport org.springframework.boot.autoconfigure.SpringBootApplication;\r\n\r\n@SpringBootApplication\r\npublic class ToysInventoryApplication {\r\n\r\n\tpublic static void main(String[] args) {\r\n\t\tSpringApplication.run(ToysInventoryApplication.class, args);\r\n\t}\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/toys_inventory/toys_inventory/src/main/java/com/example/toys_inventory/ToysInventoryApplication.java b/toys_inventory/toys_inventory/src/main/java/com/example/toys_inventory/ToysInventoryApplication.java
--- a/toys_inventory/toys_inventory/src/main/java/com/example/toys_inventory/ToysInventoryApplication.java	(revision cf075c72e27a4c4347a7ac69321b032ee5482746)
+++ b/toys_inventory/toys_inventory/src/main/java/com/example/toys_inventory/ToysInventoryApplication.java	(date 1615426089880)
@@ -6,8 +6,8 @@
 @SpringBootApplication
 public class ToysInventoryApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(ToysInventoryApplication.class, args);
-	}
+    public static void main(String[] args) {
+        SpringApplication.run(ToysInventoryApplication.class, args);
+    }
 
 }
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1615425833236)
+++ b/.idea/vcs.xml	(date 1615425833236)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/SE_MainProject_Inventory_System_WebApp.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/SE_MainProject_Inventory_System_WebApp.iml b/.idea/SE_MainProject_Inventory_System_WebApp.iml
new file mode 100644
--- /dev/null	(date 1615426080379)
+++ b/.idea/SE_MainProject_Inventory_System_WebApp.iml	(date 1615426080379)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="lib" level="project" />
+    <orderEntry type="library" name="lib1" level="project" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1615425830822)
+++ b/.idea/.gitignore	(date 1615425830822)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1615426150849)
+++ b/.idea/modules.xml	(date 1615426150849)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/SE_MainProject_Inventory_System_WebApp.iml" filepath="$PROJECT_DIR$/.idea/SE_MainProject_Inventory_System_WebApp.iml" />
+      <module fileurl="file://$PROJECT_DIR$/toys_inventory/toys_inventory/toys_inventory.iml" filepath="$PROJECT_DIR$/toys_inventory/toys_inventory/toys_inventory.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
